lib/structures/GuildChannel.js

```diff
@@ -89,7 +89,7 @@ class GuildChannel extends Channel {
     }
 
     get nsfw() {
-        return this.type !== 2 && this.name.startsWith("nsfw");
+        return this.type !== 2 && this.name.length == 4 ? this.name == "nsfw" : this.name.startsWith("nsfw-");
```

Comment:
The voice channel check is being combined with the length check, should use some parenthesis

Also use `===`